packages/calcite-components/package.json

```diff
@@ -75,14 +75,16 @@
     "sortablejs": "1.15.0"
   },
   "devDependencies": {
-    "@esri/calcite-design-tokens": "1.0.0",
+    "@esri/calcite-design-tokens": "*",
```

Comment:
The packages in the monorepo that are published to NPM need to have the actual version here for `lerna`/`release-please`, rather than `*`. Packages that aren't versioned/published (like the configs) can use `*`.